applications/serial_lte_modem/src/slm_at_tcpip.c

```diff
@@ -744,23 +759,18 @@ int handle_at_socket(enum at_cmd_type cmd_type)
 			if (err) {
 				return err;
 			}
-			err = at_params_unsigned_int_get(&at_param_list, 4, &sec_tag);
-			if (err) {
-				sec_tag = INVALID_SEC_TAG;
-			}
+			client.hostname_verify = 0;
```

Comment:
Is this really needed for regular socket?

Reply:
No this is redundant.